src/tooltip/tooltip.js

```diff
@@ -284,15 +280,16 @@ angular.module( 'ui.bootstrap.tooltip', [ 'ui.bootstrap.position', 'ui.bootstrap
             /**
              * Observe the relevant attributes.
              */
-            if (!options.useContentExp) {
-              attrs.$observe( type, function ( val ) {
+            attrs.$observe( type, function ( val ) {
+              $timeout(positionTooltip, 0, false);
+              if (!options.useContentExp) {
```

Comment:
I see now. I'll make the change. In light of that I think I also need to add a reposition to the contentExp watch, as you suggested in an earlier comment.